nemo_text_processing/text_normalization/hi/data/time/seconds.tsv

```diff
@@ -0,0 +1,60 @@
+०१	एक
```

Comment:
In Hindi, we typically use a 12-hour clock format when referring to time. To handle this properly, I created separate files for time instead of using the numbers TSV. I have also defined a specific TSV file exclusively for time.